### PR TITLE
Archana assignment8 -Banking system assignment 

### PR DESCRIPTION
**Pull request by ArchanaNJun20**

### CODE REVIEW

File: src/archanaNJun20/Bankingsystem.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
+package archanaNJun20;
+
+public class Bankingsystem {
+
+	int balance=1000;
+	int debit;
+	int credit;
+	int pBalance;
+	static int sBalance=1000;
```
Comment (KrishnaKTechnocredits) [inline src/archanaNJun20/Bankingsystem.java]: Remove unused variable.

File: src/archanaNJun20/Bankingsystem.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
+package archanaNJun20;
+
+public class Bankingsystem {
+
+	int balance=1000;
```
Comment (KrishnaKTechnocredits) [inline src/archanaNJun20/Bankingsystem.java]: No Need to initialize this variable. perform credit() method first.

File: src/archanaNJun20/Bankingsystem.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+	int credit;
+	int pBalance;
+	static int sBalance=1000;
+	static int sDebit;
+	static int sCredit;
+	static int sPBalance;
+	
+	void debitAmount(int pDebit) {
+		
+		balance=balance-pDebit;
```
Comment (KrishnaKTechnocredits) [inline src/archanaNJun20/Bankingsystem.java]: You should write the condition to verify the insufficient balance.

File: src/archanaNJun20/Bankingsystem.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+		sDebit++;
+	}
+	void creditAmount(int pCredit) {
+		balance=balance+pCredit;
+		credit++;
+		sCredit++;
+	}
+	void printBalance() {
+		pBalance++;
+		sPBalance++;
```
Comment (KrishnaKTechnocredits) [inline src/archanaNJun20/Bankingsystem.java:31]: Why are you not printing Balance in this method?

File: src/archanaNJun20/Bankingsystem.java (hunk 1 above)
Comment (ArchanaNJun20) [reply src/archanaNJun20/Bankingsystem.java]: Done

File: src/archanaNJun20/Bankingsystem.java (hunk 2 above)
Comment (ArchanaNJun20) [reply src/archanaNJun20/Bankingsystem.java]: Done

File: src/archanaNJun20/Bankingsystem.java (hunk 3 above)
Comment (ArchanaNJun20) [reply src/archanaNJun20/Bankingsystem.java]: Done

File: src/archanaNJun20/Bankingsystem.java (hunk 4 above)
Comment (ArchanaNJun20) [reply src/archanaNJun20/Bankingsystem.java:31]: done

**Review by KrishnaKTechnocredits — COMMENTED** [review]
Code looks good to me.

**Pull request merged**